tests/wallet/rpc/test_wallet_rpc.py

```diff
@@ -388,10 +388,12 @@ async def test_get_farmed_amount(wallet_rpc_environment: WalletRpcTestEnvironmen
     result = await wallet_rpc_client.get_farmed_amount()
 
     expected_result = {
+        "blocks_won": 2,
         "farmed_amount": 4_000_000_000_000,
         "farmer_reward_amount": 500_000_000_000,
         "fee_amount": 0,
         "last_height_farmed": 2,
+        "last_time_farmed": result["last_time_farmed"],  # time cannot be predicted so skipping test here
```

Comment:
can it not be predicted because we use wall clock time in the test, and not simulated time?